src/phoebus_driver.cpp

```diff
@@ -685,7 +685,7 @@ TaskListStatus PhoebusDriver::RadiationPostStep() {
     parthenon::AllReduce<bool> *pdo_gain_reducer;
     bool do_lightbulb = rad->Param<bool>("do_lightbulb");
     if (do_lightbulb) {
-      pdo_gain_reducer = rad->MutableParam<parthenon::AllReduce<bool>>("do_gain_reducer");
+      pdo_gain_reducer = rad->MutableParam<parthenon::AllReduce<bool>("do_gain_reducer");
```

Comment:
This isn't a typo. Notice that `MutableParam` contains a `<` and then `AllReduce` does. So you need two `>` at the end.

Reply:
oh wow. facepalm. ok well i will keep this open if i find something else relating to the density error. thanks

Reply:
:+1: sounds good